index.js

```diff
@@ -25,6 +25,15 @@ const moment = require('moment');
 const format = require('./momentFormat');
 const roles = require('./roles');
 
+const Consumer = require('sqs-consumer');
+const AWS = require('aws-sdk');
```

Comment:
Probably not but I kept it there for clarity since the first thing we do is override it's access & secret key. If you are truly offended it can be removed I believe 👼